index.html

```diff
@@ -354,6 +355,103 @@ <h2>Ballerina in Action</h2>
                   </div>
                </div>
             </div>
+             <div id="fourthTab" class="tab-pane fade"  >
+                 <div class="row cdisplayFlex" style="background-color: #f5f5f5;">
+                     <div class="col-lg-7 col-md-12 col-sm-12 focusPane codeBlock noBottomBorder" style="padding:20px;border: 1px #c7c7c7 solid;">
+                         <div class="FullCode" style="background-color: #f5f5f5;">
+                             <div class="highlight">
+                           <pre class="language-ballerina basic ballerina">
+                              <code class="language-ballerina ballerina" style="font-size: 12px !important;">
+import ballerina/grpc;
+
+listener grpc:Listener ep = new (9090);
+
+table&lt;Album&gt; albums = table [
+      {
+         id: "1",
+         title: "Blue Train",
+         artist: "John Coltrane",
+         price: 56.99
+      },
+      {
+         id: "2",
+         title: "Jeru",
+         artist: "Gerry Mulligan",
+         price: 17.99
+      },
+      {
+         id: "3",
+         title: "Sarah Vaughan and Clifford Brown",
+         artist: "Sarah Vaughan",
+         price: 39.99
+      }
+];
+
+@grpc:ServiceDescriptor {
+      descriptor: ROOT_DESCRIPTOR_RECORD_STORE,
+      descMap: getDescriptorMapRecordStore()
+}
+service "RecordStore" on ep {
+      remote function storeAlbum(Album album) returns Album|error {
+         albums.add(album);
+         return album;
+      }
+
+      remote function getAlbumById(string id) returns Album|error {
```

Comment:
Here, we use the generated gRPC record to create the table. Since gRPC does not create read-only records, we cannot add a key to the table. That is why we cannot access albums like this. To solve this, either we have to create a new read-only record or use query expressions to filter. What would be the better approach?

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-dev-website/pull/3728/commits/129b44a83b0653134d60907844790453b574e1c6